src/de/jost_net/JVerein/gui/view/ArbeitseinsatzUeberpruefungView.java

```diff
@@ -58,12 +59,14 @@ public void handleEvent(Event event)
     group.addLabelPair("Auswertung", aui);
 
     control.getArbeitseinsatzUeberpruefungList().paint(getParent());
-    ButtonArea buttons2 = new ButtonArea();
-    buttons2.addButton("Hilfe", new DokumentationAction(),
+    ButtonArea buttons = new ButtonArea();
+    buttons.addButton("Hilfe", new DokumentationAction(),
         DokumentationUtil.ARBEITSEINSATZ, false, "question-circle.png");
-    buttons2.addButton(control.getPDFAusgabeButton());
-    buttons2.addButton(control.getCSVAusgabeButton());
-    buttons2.addButton(butArbeitseinsaetze);
-    buttons2.paint(this.getParent());
+    buttons.addButton(control.getPDFAusgabeButton());
+    buttons.addButton(control.getCSVAusgabeButton());
+    buttons.addButton(butArbeitseinsaetze);
+    buttons.addButton("Neu", new ArbeitseinsatzAction(null), 
```

Comment:
Das ist die Frage, ob das hier wirklich Sinn macht. Die View ist ja nicht eine Liste der Arbeitseinsätze sondern der Stundensummen. Wenn man hier "Neu" hinzufügt, müsse auch ein Bearbeiten und Löschen möglich sein, das geht jedoch nicht, da ja nur die Summe angezeigt wird.

Reply:
Vor der Frage stand ich auch. Eigentlich bräuchte mann einen weiteren View der alle Arbeitseinsätze anzeigt und da gehörte es rein. Mit einem Kontextmenü beiden Einträgen für Bearbeiten und Löschen.
Ohne den hätte man wenigstens das Neu.

Soll ich den zusätzlichen View machen?
Aber wenn, dann mit einem eigenen PR.

Reply:
Ich habe das Neu wieder aus dem Arbeitseinsatz prüfen View entfernt.
Ich werde einen neuen View erstellen mit der Liste der Arbeitseinsätze.
Den bestehenden View verschiebe ich dann nach Auswertungen im Navigationsbaum. Das ist ga prktisch eine Auswertung die man in diesem View macht.
Dazu aber ein eigener PR.

Mit dem hier habe ich dann alle Kommentare eingearbeitet.